src/lib/misc/isEmptyPayload.tsx

```diff
@@ -5,5 +5,5 @@ export const isEmptyPayload = (files: UploadedFile[] | undefined) => {
     return true;
   }
   const totalBytes = files.reduce((acc, file) => acc + file.size, 0);
-  return totalBytes > 0;
+  return totalBytes === 0;
 };
```

Comment:
What is the consequence of this fix?

Reply:
Totally safe since it was determining if the payload should be preserved for debugging. In this case the check was wrongly persisting empty uploads.